DynamicAnalyzer/views/android/environment.py

```diff
@@ -40,41 +40,49 @@ def wait(self, sec):
         logger.info('Waiting for %s seconds...', str(sec))
         time.sleep(sec)
 
+    def proof_connect(self, output):
```

Comment:
Rename function name to `check_connect_error`

Reply:
Done